src/components/Task/Task.jsx

```diff
@@ -0,0 +1,26 @@
+import CloseIcon from "@mui/icons-material/Close";
+import { Div, P, Input, Button, PMark } from "./Task.styles.js";
+import { useState } from "react";
+
+export const Task = ({ text }) => {
+  const [markTask, setMarkTask] = useState(false);
```

Comment:
```suggestion
  const [isTaskMarked, setIsTaskMarked] = useState(false);
```

You'd wanna use verbs only to name functions, not variables